examples/position_ramp.py

```diff
@@ -0,0 +1,53 @@
+from ingeniamotion import MotionController
+from ingeniamotion.enums import OperationMode
+
+
+def position_ramp(final_position, mc: MotionController) -> None:
+    """
+    Creates a position ramp from current position to ``final_position`` as a ramp slope.
+
+    Args:
+        final_position: target position in counts.
+        mc: Controller with all the functions needed to perform a position ramp.
+    """
+    done = False
+    mc.motion.move_to_position(final_position)
+    while not done:
```

Comment:
This while will iterate only once, right? Am I missing something?

Reply:
No, you are right. I'm going to change it. Thanks.